.circleci/config.yml

```diff
@@ -84,14 +84,14 @@ workflows:
           orb-name: secrethub/cli
           context: publish-orb-prd
           add-pr-comment: false
-          major-release-tag-regex: '^major-v[[:digit:]]+\.[[:digit:]]+\.[[:digit:]]+$'
-          minor-release-tag-regex: '^minor-v[[:digit:]]+\.[[:digit:]]+\.[[:digit:]]+$'
-          patch-release-tag-regex: '^patch-v[[:digit:]]+\.[[:digit:]]+\.[[:digit:]]+$'
+          major-release-tag-regex: '^v[1-9][0-9]*\.0\.0+$'
+          minor-release-tag-regex: '^v[0-9]*\.[1-9][0-9]*\.0+$'
+          patch-release-tag-regex: '^v[0-9]*\.[0-9]*\.[1-9][0-9]*$'
```

Comment:
Regexes are now smarter, so we can tag it the way we always do, without a prefix.

Reply:
I tested this locally with this code: 

https://github.com/CircleCI-Public/orb-tools-orb/blob/0a48b68d679e4e720adf815935791a3ca6b20621/src/commands/dev-promote-from-git-tag.yml#L133-L142